### PR TITLE
chore(ember): Install Gember to support generating files

### PR DESCRIPTION
**Pull request by TSenter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,14 @@
   "author": "",
   "scripts": {
     "build": "pnpm --filter '@nrg-ui/*' build",
+    "format": "pnpm --filter '*' format",
+    "gember": "pnpm --filter '@nrg-ui/ember' run generate",
+    "gember:component": "pnpm --filter @nrg-ui/ember run generate component --class-based --typescript",
```
Comment (lnicely) [inline package.json:10]: This creates the component, but seems to throw this error ` ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @nrg-ui/ember@0.0.0 generate: `pnpm exec gember "component" "--class-based" "--typescript" "comp1"`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.`

When I run the command it has outlined, I get a warning that `gember` isn't a command. I've got it installed and it works per their docs, so not sure what's going on.

Comment (lnicely) [reply package.json:10]: The same happens for the other commands that use `gember` too

Comment (TSenter) [reply package.json:10]: Did you run `clean` before trying to use the commands? And can you share exactly what command you're using? I'm using `pnpm gember:component my-new-component` and it's running just fine (Node v18.20.2, pnpm v9.6.0).

Comment (TSenter) [reply package.json:10]: And make sure you're running the command from the root of the project, not the Ember package.

Comment (lnicely) [reply package.json:10]: It was the node/pnpm version causing the error. We may want to note that. Works now!

**Review by lnicely — APPROVED** [review]

**Pull request merged**